### PR TITLE
Some pages are now accessible without logging in

### PR DESCRIPTION
**Pull request by rangelkoli**
Home, FAQ and Login pages can be viewed without loggin in

+ Changed the name of the webiste from React + Vite to Travelify

### CODE REVIEW

File: frugaloo/src/App.jsx [hunk 1]
```diff
@@ -61,68 +61,79 @@ function App() {
...
+        <span className='loading loading-spinner loading-lg mr-5 text-custom-blue'></span>
         Loading Travelify..
       </div>
     );
   }
 
-  if (!session) {
-    return <Login />;
-  }
+  // if (!session) {
```
Comment (gideon-harpanahalli) [inline frugaloo/src/App.jsx]: If we dont need this code maybe we can remove this

File: frugaloo/src/components/Login.jsx [hunk 2]
```diff
@@ -2,9 +2,10 @@ import { useState, useEffect } from "react";
 import { createClient } from "@supabase/supabase-js";
 import { Auth } from "@supabase/auth-ui-react";
 
-
-const supabase = createClient('https://wqbvxqxuiwhmretkcjaw.supabase.co', 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6IndxYnZ4cXh1aXdobXJldGtjamF3Iiwicm9sZSI6ImFub24iLCJpYXQiOjE3MTk3MTYyMTQsImV4cCI6MjAzNTI5MjIxNH0.CXyPAdKKgwjmPee0OmvV4BxnQUj_4y3ARbaEuSToz6s')
-
+const supabase = createClient(
```
Comment (gideon-harpanahalli) [inline frugaloo/src/components/Login.jsx:6]: Would it be possible to get it from env vars

File: frugaloo/src/components/Navbar.jsx [hunk 3]
```diff
@@ -8,8 +8,8 @@ import addIcon from "../assets/addIcon.png";
 import { useNavigate } from "react-router-dom";
 
 const supabase = createClient(
-  "https://wqbvxqxuiwhmretkcjaw.supabase.co",
-  "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6IndxYnZ4cXh1aXdobXJldGtjamF3Iiwicm9sZSI6ImFub24iLCJpYXQiOjE3MTk3MTYyMTQsImV4cCI6MjAzNTI5MjIxNH0.CXyPAdKKgwjmPee0OmvV4BxnQUj_4y3ARbaEuSToz6s"
+  "https://srqijewflsvgaffwybfj.supabase.co/",
```
Comment (gideon-harpanahalli) [inline frugaloo/src/components/Navbar.jsx]: Here as well, if we can get it from env vars would be great!

**Comment by hemant-mistry** [comment]
@rangelkoli Can we have the popups/tooltip [whatever suits in this case] for the **Add Itinerary button** as well the **Get Started** when the user tries to make an itinerary before logging in?

image

**Review by hemant-mistry — CHANGES_REQUESTED** [review]

**Review by hemant-mistry — COMMENTED** [review]
Updated the comments

**Review by hemant-mistry — COMMENTED** [review]
Updated with comments

**Comment by rangelkoli** [comment]
> @rangelkoli Can we have the popups/tooltip [whatever suits in this case] for the **Add Itinerary button** as well the **Get Started** when the user tries to make an itinerary before logging in?
> 
> image

image
Isnt there already a tooltip for the components on the Navbar, Plus what tooltip would be used for Get Started

**Review by gideon-harpanahalli — APPROVED** [review]
LG

**Pull request closed without merging**